include/alpaka/event/EventUniformCudaHipRt.hpp

```diff
@@ -80,11 +80,11 @@ namespace alpaka
     //! The CUDA/HIP RT device event.
     template<typename TApi>
     class EventUniformCudaHipRt final
-        : public concepts::Implements<ConceptCurrentThreadWaitFor, EventUniformCudaHipRt<TApi>>
-        , public concepts::Implements<ConceptGetDev, EventUniformCudaHipRt<TApi>>
+        : public concepts::Implements<ConceptCurrentThreadWaitFor, EventUniformCudaHipRt>
+        , public concepts::Implements<ConceptGetDev, EventUniformCudaHipRt>
```

Comment:
Those two should be allowed language-wise, but there is no reason to not use the injected class name here as well.

Reply:
Looks like the CUDA/HIP compilers disagree with you here :-)